libraries/AP_BattMonitor/AP_BattMonitor_FuelLevel_Analog.cpp

```diff
@@ -58,6 +58,42 @@ const AP_Param::GroupInfo AP_BattMonitor_FuelLevel_Analog::var_info[] = {
     // @Values: -1:Not Used,11:Pixracer,13:Pixhawk ADC4,14:Pixhawk ADC3,15:Pixhawk ADC6/Pixhawk2 ADC,103:Pixhawk SBUS
     AP_GROUPINFO("FL_PIN", 43, AP_BattMonitor_FuelLevel_Analog, _pin, -1),
 
+    // @Param: FL_VLT_MAX
+    // @DisplayName: Full fuel level voltage
+    // @Description: The voltage seen on the analog pin when the fuel tank is full.
+    // @Range: 0.01 10
```

Comment:
range needs to include default value, maybe use 0 for disable, and range 0 to 10 ?

Reply:
Done